src/components/LibraryDetailPage.tsx

```diff
@@ -83,9 +91,13 @@ export class LibraryDetailPage extends React.Component<LibraryDetailPageProps, L
     return(
       <div>
         { this.renderStages() }
-        <ul id="info" className="list-group">
-          { this.renderInfo() }
-        </ul>
+        <hr></hr>
+        <div className="detail-content">
+          <Tabs items={{
+            "Basic Information": this.renderItems("basic_info"),
+            "Contact & URLs": this.renderItems("urls_and_contact"),
```

Comment:
The `renderItems` would be easier to test as a function if you also passed in the library object instead of just the key.

Reply:
How so?

Reply:
If you were to test renderItems in isolation, you cannot because it depends on `this.props` for the library data.